doi.jq

```diff
@@ -0,0 +1,186 @@
+		"alt",
+		"amazonlinux", # direct children: amazoncorretto swift
+		"api-firewall",
+		"arangodb",
+		"archlinux",
+		"bonita",
+		"centos", # direct children: eclipse-temurin ibm-semeru-runtimes percona swift
+		"clearlinux",
+		"clefos",
+		"consul",
+		"crate",
+		"debian", # direct children: adminer aerospike buildpack-deps chronograf clojure couchdb dart emqx erlang haproxy haskell hitch httpd influxdb irssi julia maven memcached mono mysql neo4j neurodebian nginx node odoo openjdk perl php postgres pypy python r-base redis rethinkdb rocket.chat ruby rust spiped swipl unit varnish
+		"eclipse-mosquitto",
+		"eggdrop",
+		"emqx",
+		"express-gateway",
+		"hitch",
+		"jobber",
+		"mageia",
+		"mono",
+		"nats",
+		"nats-streaming",
+		"oraclelinux", # direct children: mysql openjdk percona
+		"percona",
+		"photon",
+		"php", # direct children: backdrop composer drupal friendica joomla matomo mediawiki monica nextcloud phpmyadmin postfixadmin unit wordpress yourls
+		"php-zendserver",
+		"phpmyadmin",
+		"postfixadmin",
+		"r-base",
+		"rocket.chat",
+		"rockylinux",
+		"sl",
+		"spiped",
+		"teamspeak",
+		"traefik",
+		"ubuntu", # direct children: buildpack-deps couchbase eclipse-temurin elasticsearch gazebo gradle ibmjava ibm-semeru-runtimes kibana kong logstash mariadb mongo neurodebian odoo php-zendserver rabbitmq ros sapmachine silverpeas swift
+		"varnish",
+		"vault",
+		"yourls",
+		"znc",
+		"adminer",
+		"aerospike",
+		"amazoncorretto", # direct children: jetty maven tomcat
+		"backdrop",
+		"buildpack-deps", # direct children: erlang gcc golang haskell haxe influxdb kapacitor node openjdk perl pypy python rakudo-star ruby rust telegraf
+		"composer", # direct children: drupal
+		"couchbase",
+		"dart",
+		"eclipse-temurin", # direct children: cassandra clojure flink gradle groovy jetty jruby lightstreamer liquibase maven neo4j orientdb solr sonarqube spark storm tomcat tomee unit zookeeper
+		"erlang", # direct children: elixir
+		"friendica",
+		"gazebo",
+		"groovy",
+		"haskell",
+		"haxe",
+		"ibm-semeru-runtimes", # direct children: maven open-liberty tomee websphere-liberty
+		"ibmjava", # direct children: maven websphere-liberty
+		"jetty", # direct children: geonetwork
+		"joomla",
+		"kong",
+		"lightstreamer",
+		"liquibase",
+		"matomo",
+		"mediawiki",
+		"monica",
+		"neurodebian",
+		"nextcloud",
+		"node", # direct children: express-gateway ghost mongo-express unit
+		"odoo",
+		"open-liberty",
+		"orientdb",
+		"python", # direct children: hylang plone satosa unit
+		"rakudo-star",
+		"ros",
+		"sapmachine", # direct children: maven
+		"satosa",
+		"silverpeas",
+		"spark",
+		"storm",
+		"swift",
+		"tomcat", # direct children: convertigo geonetwork xwiki
+		"tomee",
+		"websphere-liberty",
+		"clojure",
+		"convertigo",
+		"geonetwork",
+		"maven",
+		"plone",
+
+		empty
+	]
+;
+
+# a helper for "build_should_sbom"
+def _sbom_subset:
```

Comment:
Do we need a check to ensure that something in `_sbom_subset` is not in `repos_anti_subset`?

Reply:
:shrug:

I thought about it, but didn't worry about it too much because ideally, we work to shrink `repos_anti_subset` to zero over time, and they are somewhat unrelated concerns (it could be valid for us to have a case of something we've validated the SBOM for, but aren't ready to move yet).

Reply:
In other words, I'd love for this to be the source-of-truth for @LaurentGoderre's validation work, and _not_ block that on our ability to add new things. :eyes:

Reply:
Oh, good point. 👍